conference/react-js/src/containers/SignUpForm/SignUpForm.jsx

```diff
@@ -15,27 +15,29 @@ class SignUp extends Component {
       mail: '',
       password: '',
       confirm: '',
+      passLen: false,
       isRegistrated: true,
       responseOk: false,
       responseStatus: 0,
     };
+  }
 
-    this.formChangeHandler = (e) => {
+    formChangeHandler = (e) => {
       const target = e.target;
 
       this.setState({
         [target.name]: target.value,
       });
     };
 
-    const registration = ({ mail, password, confirm, fname, lname }) => {
+    registration = ({ mail, password, confirm, fname, lname }) => {
```

Comment:
the same here. `fname` and `lname` may be unclear for sb

Reply:
lname and fname are those we need to send to backend in the body of our request (same listed on wiki)